app/utils/requests/collaborationQuestions/requests.ts

```diff
@@ -54,8 +57,8 @@ export async function getCollaborationQuestionsByProjectId(projectId: string) {
 
 export async function getBasicCollaborationQuestionById(id: string): Promise<BasicCollaborationQuestion | undefined> {
   try {
-    const response = await strapiGraphQLFetcher(GetCollaborationQuestionByIdQuery, { id });
-    const data = response.collaborationQuestion?.data;
+    const response = await strapiGraphQLFetcher(GetCollaborationQuestionByIdQuery, { documentId: id });
+    const data = response.collaborationQuestion as ResultOf<typeof CollaborationQuestionFragment>;
```

Comment:
cast is not necessary
```suggestion
    const data = response.collaborationQuestion
```